index.html

```diff
@@ -5,8 +5,39 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Alex Portfolio</title>
+    
+    <!--Style-->
+    <link rel="stylesheet" href="./style/style.css"> 
+
+    <!--Roboto-->
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <!--Crete round-->
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <!--Crete round-->
+    <link href='https://fonts.googleapis.com/css2?family=Poppins' rel='stylesheet'>
+
 </head>
 <body>
-    
+    <header>
+
+        <nav>
+            <ul>
+                <li class="my-logo">Alex Dev</li>
```

Comment:
- [ ] The logo should be a clickable link. Kindly use an `<a>`  tag.
- [ ] As you can see here 
![image](https://user-images.githubusercontent.com/43698511/129241067-9a8354c8-8db7-488e-8730-82f83e4bddfd.png)
The logo and the menu button are aligned. 
![image](https://user-images.githubusercontent.com/43698511/129241442-62f91791-990c-46a0-9952-a36104f14ea7.png)

Your header looks a bit different kindly fix the issue.

Reply:
- Logo now is a clickable link
- Logo and menu aligned
- List of Icons added
- Margins corrected of content
- Style improvements of code readability